loader/src/main/java/net/neoforged/fml/loading/TransformerDiscovererUtils.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.fml.loading;
+
+import cpw.mods.jarhandling.JarContentsBuilder;
+import cpw.mods.jarhandling.JarMetadata;
+import cpw.mods.jarhandling.SecureJar;
+
+import java.nio.file.Path;
+
+public class TransformerDiscovererUtils {
+
+
+    public static boolean shouldLoadInServiceLayer(Path... path) {
```

Comment:
I'd put it in the same class as `TransformerDiscovererConstants`, potentially renaming it.